Orange/widgets/data/owrank.py

```diff
@@ -23,47 +23,19 @@ def table(shape, fill=None):
     return [[fill for j in range(shape[1])] for i in range(shape[0])]
 
 
-_score_meta = namedtuple(
-    "_score_meta",
+score_meta = namedtuple(
+    "score_meta",
     ["name",
      "shortname",
-     "score",
-     "supports_regression",
-     "supports_classification",
-     "handles_discrete",
-     "handles_continuous"]
+     "score"]
 )
 
-
-class score_meta(_score_meta):
-    # Add sensible defaults to __new__
-    def __new__(cls, name, shortname, score,
-                supports_regression=True, supports_classification=True,
-                handles_continuous=True, handles_discrete=True):
-        return _score_meta.__new__(
-            cls, name, shortname, score,
-            supports_regression, supports_classification,
-            handles_discrete, handles_continuous
-        )
-
 # Default scores.
 SCORES = [
-    score_meta(
-        "Information Gain", "Inf. gain", score.InfoGain,
-        supports_regression=False,
-        supports_classification=True,
-        handles_continuous=False,
-        handles_discrete=True),
-    score_meta(
-        "Gain Ratio", "Gain Ratio", score.GainRatio,
-        supports_regression=False,
-        handles_continuous=False,
-        handles_discrete=True),
-    score_meta(
-        "Gini Gain", "Gini", score.Gini,
-        supports_regression=False,
-        supports_classification=True,
-        handles_continuous=False),
+    score_meta("Information Gain", "Inf. gain", score.InfoGain),
+    score_meta("Gain Ratio", "Gain Ratio", score.GainRatio),
+    score_meta("Gini Gain", "Gini", score.Gini),
+    score_meta("ReliefF", "ReliefF", score.ReliefF),
```

Comment:
There are also some scoring methods from sklearn (chi2, anova, ...) missing. I'm not sure which do we want in the widget, probably not all of them.